tests/e2e/send-message.js

```diff
@@ -410,22 +412,7 @@ describe('Send message', () => {
 
       common.goToMessages();
 
-      const liIdentifier = messageInList(ALICE._id);
-      helper.waitUntilReady(element(by.css(liIdentifier)));
-      element(by.css(liIdentifier + ' a')).click();
-
-      helper.waitForAngularComplete();
-
-      browser.wait(() => {
-        const el = element(by.css('#message-header .name'));
-        return helper
-          .getTextFromElement(el)
-          .then(text => text === 'Unknown sender')
-          .catch(err => {
-            // item may have been removed from the page when the RHS is refreshed
-            console.log('Caught and ignoring an error trying to getText', err);
-          });
-      }, 12000);
+      clickLhsEntry(ALICE._id, 'Unknown sender');
```

Comment:
Reuse function in attempt to make this test more reliable. It didn't work (I fixed it elsewhere) but it's cleaner anyway so I left it in.